learn_evm/evm_opcodes.md

```diff
@@ -10,184 +10,188 @@ The gas information is a work in progress. If an asterisk is in the Gas column,
+| [`0xf3`](#return)         | RETURN         | Halt execution returning output data                                                                                                                         | -                                                                                                         | 0           |
+| [`0xf4`](#delegatecall)   | DELEGATECALL   | Message-call into this account with an alternative account's code, but persisting into this account with an alternative account's code                       | -                                                                                                         | Complicated |
+| [`0xf5`](#create2)        | CREATE2        | Create a new account and set creation address to `sha3(sender + sha3(init code)) % 2**160`                                                                   | -                                                                                                         |
+| `0xf6` - `0xf9`           | Unused         | -                                                                                                                                                            | -                                                                                                         |
+| [`0xfa`](#staticcall)     | STATICCALL     | Similar to CALL, but does not modify state                                                                                                                   | -                                                                                                         | 40          |
+| `0xfb`                    | Unused         | -                                                                                                                                                            | -                                                                                                         |
+| [`0xfd`](#revert)         | REVERT         | Stop execution and revert state changes, without consuming all provided gas and providing a reason                                                           | -                                                                                                         | 0           |
+| `0xfe`                    | INVALID        | Designated invalid instruction                                                                                                                               | -                                                                                                         | 0           |
+| [`0xff`](#selfdestruct)   | SELFDESTRUCT   | Halt execution and register account for later deletion                                                                                                       | -                                                                                                         | 5000\*      |
```

Comment:
Lots of whitespace adjustments here, looks good, did you use a vim macro or something else to take care of this?
One consideration: if this content is updated later, would it be any harder to keep formatting consistent now that things are more precisely aligned?

Reply:
This was done through prettier: `prettier **/*.md --write`. We could include this as part of the standard workflow or CI.
The formatting on solidity code was done using `forge fmt **/*.sol`. It might be better to stick to one tool, but I think prettier didn't have native solidity support. I can look more into the options however.